quantifier/src/main/java/edu/illinois/cs/cogcomp/quant/features/POSBigrams.java

```diff
@@ -0,0 +1,29 @@
+/**
+ * This software is released under the University of Illinois/Research and Academic Use License. See
+ * the LICENSE file in the root folder for details. Copyright (c) 2016
+ *
+ * Developed by: The Cognitive Computation Group University of Illinois at Urbana-Champaign
+ * http://cogcomp.cs.illinois.edu/
+ */
+package edu.illinois.cs.cogcomp.quant.features;
+
+import edu.illinois.cs.cogcomp.core.datastructures.textannotation.Constituent;
+import edu.illinois.cs.cogcomp.edison.features.Feature;
+import edu.illinois.cs.cogcomp.edison.features.factory.SpanFeaturesOrdered;
+import edu.illinois.cs.cogcomp.edison.utilities.EdisonException;
+
+import java.util.LinkedHashSet;
+import java.util.Set;
+
+public class POSBigrams extends LBJavaFeatureExtractor {
```

Comment:
A number of these feature extractors look like they may duplicated existing extractors in Edison.
The best way is probably to merge as is, then create an issue; probably requires some effort & testing to make sure there are no behavioral changes.